src/diet/index.ts

```diff
@@ -238,10 +242,14 @@ class DietPlanner {
     this.seasoning = menu.find(
       (item) => item.item === $item`Special Seasoning`
     );
-    this.mayoflex =
-      getWorkshed() === $item`portable Mayo Clinic`
-        ? menu.find((item) => item.item === $item`Mayoflex`)
-        : undefined;
+    this.mayoLookup = new Map<Item, MenuItem>();
+    if (mayoInstalled()) {
+      [Mayo.flex, Mayo.zapine].forEach((mayo) => {
```

Comment:
Minor but you're really doing a `find` here not a `forEach` here...

Reply:
I think this is more of a for but we tend to use for... of instead of forEach.